games-rpg/runescape-launcher/runescape-launcher-2.2.4-r3.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit unpacker xdg gnome2-utils
+
+DESCRIPTION="Official RuneScape NXT client launcher"
+HOMEPAGE="http://www.runescape.com"
+
+SRC_URI="http://content.runescape.com/downloads/ubuntu/pool/non-free/r/${PN}/${PN}_${PV}_amd64.deb"
+
+QA_PREBUILT="/opt/runescape-launcher/runescape"
+
+SLOT="0"
+
+IUSE="kde"
+
+KEYWORDS="-* ~amd64"
+
+LICENSE="RuneScape-EULA"
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}"
+
+RDEPEND="
+	media-libs/libpng:1.2
+	>=media-libs/libsdl2-2.0.2
+	media-libs/glew:1.10
+	>=media-libs/libvorbis-1.3.2
+	>=net-misc/curl-7.35.0
+"
+DEPEND=""
+
+src_prepare() {
+	# Fix path in launcher script
+	sed -i "s:/usr/share/games/$PN:/opt/$PN:" usr/bin/$PN || die
```

Comment:
As a separate cleanup commit, could you replace all `$PN`s etc. with `${PN}` (i.e. always use `${...}` form)?